mapstruct-2/src/main/java/com/baeldung/mapstruct/date/mapper/UserMapper.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.mapstruct.date.mapper;
+
+import com.baeldung.mapstruct.date.model.User;
+import com.baeldung.mapstruct.date.model.UserDTO;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@Mapper
+public interface UserMapper {
```

Comment:
This is not tested

Reply:
Wrote the UT but failed to add it in the commit. Will fix it later